Pokemon Unity/Assembly-CSharp.csproj

```diff
@@ -57,10 +57,16 @@
     <Compile Include="Assets\Scripts2\Enums\Pokemons.cs" />
     <Compile Include="Assets\Scripts2\Enums\Items.cs" />
     <Compile Include="Assets\Scripts2\Enums\Moves.cs" />
+    <Compile Include="Assets\Scripts2\Enums\SaveEvent.cs" />
     <Compile Include="Assets\Scripts2\EventHandlers\BattlePokemonHandler.cs" />
     <Compile Include="Assets\Scripts2\Enums\Enums.cs" />
     <Compile Include="Assets\Scripts2\Pokemon\Type.cs" />
     <Compile Include="Assets\Scripts2\Pokemon\MoveDatabase.cs" />
+    <Compile Include="Assets\Scripts2\Saving\SaveEvent.cs" />
+    <Compile Include="Assets\Scripts2\Saving\EventListener.cs" />
+    <Compile Include="Assets\Scripts2\Saving\GlobalSaveManager.cs" />
+    <Compile Include="Assets\Scripts2\Saving\SaveData.cs" />
+    <Compile Include="Assets\Scripts2\Saving\Location\SerializeVector3.cs" />
```

Comment:
If you cherry pick, you'll have to be sure to build the project again; because these will get deleted.